python-package/lightgbm/engine.py

```diff
@@ -277,18 +283,45 @@ class CVBooster:
         The best iteration of fitted model.
     """
 
-    def __init__(self):
+    def __init__(
+        self,
+        model_file: Optional[Union[str, Path]] = None
+    ):
         """Initialize the CVBooster.
 
         Generally, no need to instantiate manually.
```

Comment:
Is this statement still valid?..

Reply:
+1 I think this line should be removed. Since the methods introduced in this PR are instance methods and not class methods.

Reply:
The statement has been removed in 
[cf52d45](https://github.com/microsoft/LightGBM/pull/5160/commits/cf52d45098621771de0f6d5bed3c4fc835af4d3a).